cmd/k3s/main.go

```diff
@@ -100,6 +101,30 @@ func findDataDir() string {
 	return dataDir
 }
 
+// findPreferBundledBin searches for prefer-bundled-bin from the CLI args and config file.
+func findPreferBundledBin(args []string) bool {
+	preferRes := configfilearg.MustFindString(args, "prefer-bundled-bin")
+	for i, arg := range args {
+		flagName := "--prefer-bundled-bin"
```

Comment:
I don't think we need to set this every time through the loop, do we?
```suggestion
	flagName := "--prefer-bundled-bin"
	for i, arg := range args {
```

Reply:
Fixed